src/reducers/comments.spec.tsx

```diff
@@ -0,0 +1,168 @@
+import reducer, {
+  CommentInfo,
+  actions,
+  createCommentsKey,
+  createEmptyCommentInfo,
+  createInternalComment,
+} from './comments';
+import { createInternalVersion } from './versions';
+import { createFakeExternalComment, fakeVersion } from '../test-helpers';
+
+describe(__filename, () => {
+  const createCommentInfo = (info: Partial<CommentInfo> = {}): CommentInfo => {
+    return {
+      ...createEmptyCommentInfo(),
+      ...info,
+    };
+  };
+
+  describe('beginComment', () => {
+    it('begins a comment by key', () => {
+      const fileName = 'manifest.json';
+      const line = 123;
+      const versionId = 1;
+
+      const state = reducer(
+        undefined,
+        actions.beginComment({ versionId, fileName, line }),
```

Comment:
Nit: This isn't a big deal (hence the `Nit:`) but I notice that the order of the properties in the calls to the various actions, as well as in the calls to `createCommentsKey` are inconsistent throughout the file. Sometimes `versionId` is first and sometimes `fileName` is first. It really doesn't matter, but I personally try to keep things consistent by always putting them in alphabetical order. I'm just calling this out to bring it to your attention in case you didn't realize and would like to fix it, but if you don't think it's worth fixing I'm fine with that too.